packages/ui-extensions/src/surfaces/admin/components/UnorderedList/examples/store-setup-checklist.html

```diff
@@ -0,0 +1,32 @@
+<s-stack gap="base">
+  <s-box borderWidth="small-100" borderRadius="base" padding="base">
+    <s-unordered-list>
+      <s-list-item>Free shipping on orders over $50</s-list-item>
```

Comment:
Is this list intended to be part of the "store-setup-checklist" example? Seems like a separate topic than the following lists.

<img width="632" height="462" alt="Image" src="https://github.com/user-attachments/assets/b9623709-aea2-416c-a47b-312dbd14a553" />